File: app/src/main/java/com/stepyen/xlearn/fragment/utils/launch/BActivity.java
package com.stepyen.xlearn.fragment.utils.launch;

import android.content.Intent;
import android.util.Log;
import android.view.View;
import android.widget.TextView;
import com.stepyen.xlearn.base.BaseTestActivity;

/**
 * date：2019/4/4
 * author：stepyen
 * description：
 */
public class BActivity extends BaseTestActivity {
    private static final String TAG = "BActivity";

    @Override
    public void initView() {
        Log.i(TAG, "onCreate: ");

        TextView tv = new TextView(this);
        tv.setText("BActivity");
        addView(tv);

        addView("CActivity", new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(mActivity, CActivity.class));
            }
        });
    }

    @Override
    protected void onNewIntent(Intent intent) {
        super.onNewIntent(intent);
        Log.i(TAG, "onNewIntent: ");
    }

    @Override
    protected void onStart() {
        super.onStart();
        Log.i(TAG, "onStart: ");
    }

    @Override
    protected void onResume() {
        super.onResume();
        Log.i(TAG, "onResume: ");
    }

    @Override
    protected void onPause() {
        super.onPause();
        Log.i(TAG, "onPause: ");
    }

    @Override
    protected void onStop() {
        super.onStop();
        Log.i(TAG, "onStop: ");
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        Log.i(TAG, "onDestroy: ");
    }

    @Override
    protected void onRestart() {
        super.onRestart();
        Log.i(TAG, "onRestart: ");
    }
}

File: app/src/main/java/com/stepyen/xlearn/activity/module/task/BActivity.kt
package com.stepyen.xlearn.activity.module.task

import android.content.Intent
import android.view.View
import com.stepyen.common.BaseLifePageActivity
import com.stepyen.xlearn.base.BasePageActivity

/**
 * date：2020-03-13
 * author：stepyen
 * description：
 *
 */
class BActivity : BaseLifePageActivity() {
    override var TAG =  "B_TAG"

    override fun initView() {
        addTextView("当前是 B")
        addButton("打开 C",View.OnClickListener {
            startActivity(Intent(this@BActivity,CActivity::class.java).apply {

            })
        })
    }

}